### ISSUE TITLE
Autocompletions should not be triggered when inserting a newline

### ISSUE THREAD

**Issue by its-dhung** [spyder-ide/spyder#10783]
## Problem Description
Autocompletions are triggered when inserting a newline. In many cases, the user just wants to move a line of code to the next line instead of editing the source code.

### What steps reproduce the problem?

1. Start with the file
```python
foo = 'bar'
```
2. Position your cursor in front of `f` on the first line.
3. Hit enter to move the line to the next line.

### What is the expected output? What do you see instead?
__Expected:__ The line is moved with no other side effects.

__Actual:__ The line is moved and then autocompletions are triggered, showing these faulty completions from Jedi.

![image](https://user-images.githubusercontent.com/3752225/69291141-22dc3900-0bb7-11ea-9660-d0a9b2bd4593.png)

This happens regardless of the value of the `automatic_completions_after_chars` setting.

## Versions

Python 3.7.0 64-bit | Qt 5.9.6 | PyQt5 5.9.2 | Darwin 19.0.0 

### Dependencies

```
cloudpickle >=0.5.0          :  1.2.2 (OK)
pygments >=2.0               :  2.4.2 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
nbconvert >=4.0              :  5.6.1 (OK)
sphinx >=0.6.6               :  2.2.1 (OK)
pylint >=0.25                :  2.4.4 (OK)
psutil >=0.3                 :  5.6.5 (OK)
qtawesome >=0.5.7            :  0.6.0 (OK)
qtpy >=1.5.0                 :  1.9.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
zmq >=17                     :  18.1.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
numpydoc >=0.6.0             :  0.9.1 (OK)
spyder_kernels >=1.8.0;<2.0.0:  1.8.0 (OK)
qdarkstyle >=2.7             :  2.7 (OK)
atomicwrites >=1.2.0         :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
watchdog                     :  None (OK)
keyring                      :  None (OK)
pexpect >=4.4.0              :  4.7.0 (OK)
pympler                      :  None (OK)
sympy >=0.7.3                :  None (NOK)
cython >=0.21                :  None (NOK)
IPython >=4.0                :  7.9.0 (OK)
matplotlib >=2.0.0           :  None (NOK)
pandas >=0.13.1              :  None (NOK)
numpy >=1.7                  :  None (NOK)
scipy >=0.17.0               :  None (NOK)
pyls >=0.31.0;<0.32.0        :  0.31.1 (OK)
applaunchservices >=0.1.7    :  0.2.1 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
```

**Comment by ccordoba12** [comment]
@andfoy, what do you think about this one?

**Comment by ccordoba12** [comment]
I think this was fixed by PR #10802.
